hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -84,11 +88,33 @@ public void init() throws ServletException {
         OMConfigKeys.OZONE_DB_CHECKPOINT_TRANSFER_RATE_KEY,
         OMConfigKeys.OZONE_DB_CHECKPOINT_TRANSFER_RATE_DEFAULT);
 
+    // TODO: Add servlet
+//    getServletContext().addServlet(OMDBCheckpointAuthorizedServlet);
+
     if (transferBandwidth > 0) {
       throttler = new DataTransferThrottler(transferBandwidth);
     }
   }
 
+  private boolean checkAcls(String username) {
```

Comment:
NIT: For me the name is confusing. We check the admin permission not the ACLs. (But it's just a personal opinion)

Reply:
You're right. renamed to `hasPermission`.